src/scitokens/scitokens.py

```diff
@@ -74,13 +89,42 @@ def verify(self):
         raise NotImplementedError()
 
 
-    def serialize(self, include_key=False):
+    def serialize(self, include_key=False, issuer = None, lifetime = 600):
         """
         Serialize the existing SciToken.
+        
+        :return str: base64 encoded token
         """
         
+        if self._key == None:
+            raise MissingKeyException("Unable to serialize, missing private key")
+        
+        # Issuer needs to be available, otherwise throw an error
+        if issuer == None and 'iss' not in self._claims:
+            raise MissingIssuerException("Issuer not specific in claims or as argument")
+        
+        if not issuer:
+            issuer = self._claims['iss']
+        
+        issue_time = int(mktime(datetime.utcnow().timetuple()))
```

Comment:
Allow these claims to be set externally - only set them if they are not already present.

Further, change to `time.time()`?